3rdlib/quazip/quazip.pro

```diff
@@ -10,6 +10,8 @@ CONFIG += staticlib
 
 INCLUDEPATH += ../zlib
 
+QMAKE_CXXFLAGS += -std=c++11
+
```

Comment:
It does not compile fine on some compilers. With Clang 4.0.0 on Ubuntu, it does not work for me. I probably haven't noticed this until now because I use other compilers (ones that default to C++11 maybe?). I'm getting some errors with QuaZIP about "QAtomic", which probably requires the `std::atomic` type that is only found in C++11+, and this change fixes everything. I will try to push it upstream after some more testing.